<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="step">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_iteration">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reStart">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Re-start</string>
    </property>
   </widget>
   <widget class="QSlider" name="grid_size_slider">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>130</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_density">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iterate">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>240</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_iterations">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>240</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Iterations</string>
    </property>
   </widget>
   <widget class="MyWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="density_slider">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_iterations_text">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Iteration(s):</string>
    </property>
   </widget>
   <widget class="QLabel" name="grid_size_text">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_density_text">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Density:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_grid_size">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyWidget</class>
   <extends>QWidget</extends>
   <header>mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
